source/change-streams/tests/unified/change-streams.yml

```
@@ -177,3 +177,163 @@ tests:
                 comment: "comment"
               commandName: getMore
               databaseName: *database0
+
+  - description: "Test unknown operationType MUST NOT err"
+    runOnRequirements:
+      - minServerVersion: "3.6"
+    operations:
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          # using $project to simulate future changes to ChangeStreamDocument structure
+          pipeline: [ { $project: { operationType: "addedInFutureMongoDBVersion", ns: 1 } } ]
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: insertOne
+        object: *collection0
+        arguments:
+          document: {
+            "_id": 1,
+            "a": 1
+          }
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult: {
+          "operationType": "addedInFutureMongoDBVersion",
+          "ns": {
+            "db": "database0",
+            "coll": "collection0"
+          }
+        }
+
+  - description: "Test newField added in response MUST NOT err"
+    runOnRequirements:
+      - minServerVersion: "3.6"
+    operations:
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          # using $project to simulate future changes to ChangeStreamDocument structure
+          pipeline: [ { $project: { operationType: 1, ns: 1, newField: "newFieldValue" } } ]
+        saveResultAsEntity: &changeStream0 changeStream0
```

```suggestion
        saveResultAsEntity: *changeStream0
```

No need to redefine the anchor unless you expect this test might be copied to another file. That said, YAML also allows redefinition (last one wins) so feel free to ignore this suggestion.